<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iridium Archive</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/explorer.ico</normaloff>:/explorer.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolButton" name="parentPath">
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="filePath">
       <property name="placeholderText">
        <string>Path</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fileFilter">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTreeView" name="fileList">
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
